src/appMain/CMakeLists.txt

```diff
@@ -196,12 +222,25 @@ endif ()
 
 # Install rules
 install(TARGETS ${PROJECT} DESTINATION bin)
+
 install(
   FILES log4cxx.properties audio.8bit.wav test.txt smartDeviceLink.ini
-    hmi_capabilities.json sdl_preloaded_pt.json
+    hmi_capabilities.json
```

Comment:
@dev-gh it is related because sdl_preloaded_pt.json is installed later in next block of code with RC check.